tests/Core/ArcTable.Tests.fs

```diff
@@ -1623,6 +1623,46 @@ let private tests_AddColumns =
         ]
     ]
 
+
+//New Test for AddColumnFill
+let private tests_AddColumnFill = 
+    testList "AddColumnFill" [ 
+        testCase "addColumnFill to preexisting table with rows" (fun () ->  // AddColumnFill to Table with Rows // create ftestCase which prioritizes the testCase but only on local!!
+            let table = create_testTable()
+            let expectedValue = CompositeCell.createUnitizedFromString("1")
```

Comment:
Rename `expectedValue` to `cell`. This has twofold reason:
- `value` is more often used for values of primitive type
- even though the cell is also used as the `expected` value for the `equal` call later. It's primary use is for being used in the `AddColumnFill` method